packages/core/src/components/date-picker/date-picker.stories.ts

```diff
@@ -8,6 +10,14 @@ import {
 } from './date-picker.template';
 import readme from './readme.md';
 
+function decorator(story: ArgsStoryFn<TemplateResult>): TemplateResult {
+  return html`
+    <div id="date-picker-container-mock" style="width: 175px;">
```

Comment:
Is dit `id` ergens voor nodig, of komt dit uit een copy/paste actie vandaan?

Reply:
Dat laatste.